Source/UI/Authentication/LimeAuthAuthenticationUI.swift

```diff
@@ -228,7 +229,10 @@ public extension LimeAuthAuthenticationUI {
         var uiRequest = Authentication.UIRequest()
         uiRequest.prompts.activityMessage = opStrings.removeDevice_Activity
         uiRequest.prompts.successMessage = opStrings.removeDevice_Success
-        uiRequest.options = [ .usePossession, .allowBiometryFactor ]
+        
+        if session.hasBiometryFactor && PA2Keychain.canUseBiometricAuthentication {
+            uiRequest.options.insert(.allowBiometryFactor)
+        }
```

Comment:
The `.allowBiometryFactor` is just configuration that the biometry factor can be used. The operation executor has to decide whether it's really possible to sign with biometry or not. Check closed issue #16 and `sanitizeRequestOptions()` function, which is I think the best place where to fix this.

Reply:
ok, moved to executor